variables.tf

```diff
@@ -26,25 +50,37 @@ variable "read_capacity_units" {
   default     = 5
 }
 
-variable "table_encryption" {
-  description = "Server side table encryption at rest. i.e. true | false"
+variable "stream_enabled" {
+  description = "Enable the stream setting on the table."
   type        = "string"
-  default     = true
+  default     = "false"
```

Comment:
Same statement as previous. Should be more of a bool string: `false` without quotes. Would probably updates references in tests as well.

Reply:
Fixed this one in next commit